root_path_no_dot.oval

```diff
@@ -0,0 +1,49 @@
+---
+documentation_complete: true
+title: Ensure that No Dangerous Directories Exist in Root's Path
+primary_id: root_path_no_dot
+check_type: compliance
+family: unix
+version: 1
+platforms:
+  - Red Hat Enterprise Linux 7
+  - multi_platform_fedora
+
+description: |
+  The environment variable PATH should be set correctly for the root user.
+
+operation:
+  - and: begins_colon_period, begins_doublecolon
```

Comment:
Make this into a list?

Reply:
or could we use `all:` and `any:`?

Reply:
@jamescassell probably would have something like:
```complex: (something and something) or something else```

@jamescassell is your vision for all and any?

Reply:
Do we have to reinvent this part at all? I think it's better to stick to OVAL nomenclature to make it easier to translate ideas. What about this?

```
criteria:
  - and:
    - a
    - b
    - or:
      - c
      - d
  - c
```

Reply:
@dahaic how would you handle the following then:
```
- and:
    - a
    - or:
      - c
      - d
    - or:
      - e
      - f
```

Reply:
@redhatrises I am confused, yaml can handle that just fine:

```
[{'and': ['a', {'or': ['c', 'd']}, {'or': ['e', 'f']}]}]
```